amivapi/settings.py

```diff
@@ -127,6 +127,7 @@
     'event please click this link to free your spot for someone else:\n'
     '\n{link}\n'
     'You cannot sign out of this event after {deadline}.\n\n'
+    '{additional_info_text}\n\n'
```

Comment:
This template might result in a very large distance between the deadline and the "Best Regards" phrase. We should somehow ensure that `\n\n` after `{additinal_info_text}` is not added if the `additional_info_text` is an empty string.

The following options might be sensible:
- we provide two different templates for those cases.
- we use the templating engine included with flask (eve is based on flask). It is easy to use as seen in one of our other projects [1].

Aside from that, we might think about providing HTML content in addition to the plaintext version so that the email looks more fancy. But this should be handled in a separate PR to minimize the changes induced by a single PR, which makes it easier to review.

[1] https://gitlab.ethz.ch/amiv/kontakt/website-backend/-/blob/master/kontaktweb/mail.py?ref_type=heads#L23